run-tests.php

```diff
@@ -1062,7 +1062,7 @@ function error_report($testname, $logname, $tested)
 	}
 }
 
-function system_with_timeout($commandline, $env = null, $stdin = null)
+function system_with_timeout($commandline, $env = null, $stdin = null, $captureStdOut = true, $captureStdErr = true)
```

Comment:
It's a nice approach, but i'd rather avoid adding the new section to run-tests.php. It is not a general case and probably only useful for this concrete case, so IMO a new section is not justified.

What could be done instead, were doing an extra PHP run inside the test. Fe we do this for the CLI server, you can cehck sapi/cli/tests/php_cli_server.inc for a simple example framework. It is also better to be close to the real word, when testing redirection. Like

`php.exe -r "if (stream_vt100_support(STDOUT, true)) echo \"\033[101;93msupported\033[0m\n\"; else echo 'not supported'; " > tmp`

should should out "not supported" into a file on both linux and windows. Or i don't get this part right?

Thanks.